src/app.ts

```diff
@@ -33,6 +34,14 @@ const main = async () => {
 
   server.applyMiddleware({ app });
 
+  async function setupArticleRefreshCron() {
+    // Refresh articles every minute (testing purposes - will be 12 hours later)
+    cron.schedule('* * * * *', async () => {
```

Comment:
Maybe instead of using a 3rd party service like Zapier just to ping a few RSS feeds, we can do a conditional GET request and then choose to parse every minute. I don't believe just doing a simple GET request for the last refresh date is too much overhead. 

It is too much overhead to be parsing each RSS feed every minute though (it's ~130 articles every minute that are read and put in / updated).